templates/partials/technology-card.html

```diff
@@ -0,0 +1,32 @@
+{% set array_length = section_title|length %}
+<div class="container-fluid content-section">
+    <div class="row mt-5">
+        <div class="container-fluid w-75">
+            <div class="row">
+                <h3><a id="technology">{{ title }}</a></h3>
+                <p class="lead-in-text">{{ lead_in_text }}</p>
+
+            <div>
+                <div class="row">
+                    <div class="d-flex flex-column flex-md-row ">
+                        {% for array_item in range(array_length) %}
+                            <div class="d-flex flex-column technology-card">
+                                <h6 class="technology-title">{{ section_title[array_item] }}</h6>
```

Comment:
I think the title is bolded in figma but not here

Reply:
The font weight and font size are the same as on Figma.  But I did remove the Helvetica Neue  font family because of the issues it caused.